app/helpers/app_groups_helper.rb

```diff
@@ -19,4 +19,14 @@ def set_role_buttons(app_group_user, roles = {}, opts = {})
       end
     end
   end
+
```

Comment:
why do we put these authorization logic here? why not put this in policy+controller like the other?

Reply:
moved the logic to policy